core/modules/node/node.module

```diff
@@ -1658,15 +1658,7 @@ function node_menu() {
     'menu_name' => 'management',
     'weight' => -20,
   );
-  $items['rss.xml'] = array(
-    'title' => 'RSS feed',
-    'page callback' => 'node_feed',
-    'access arguments' => array('access content'),
-    'type' => MENU_CALLBACK,
-    // Pass a FALSE and array argument to ensure that additional path components
-    // are not passed to node_feed().
-    'page arguments' => array(FALSE, array()),
-  );
```

Comment:
Hm, for sites upgrading from previous versions of Backdrop/Drupal, they won't have the `rss.xml` view activated right? So we might need to keep this to keep that page from breaking until 2.x?